### PR TITLE
enable more eslint rules

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

File: src/series/graph.serie.line.js [hunk 1]
```diff
@@ -1703,10 +1702,10 @@ class SerieLine extends Serie {
 
     switch ( this.getStyle( selectionType ).lineStyle ) {
 
-      case 2:
+      case 1:
```
Comment (targos) [inline src/series/graph.serie.line.js]: cc @NPellet just to make sure it is a bug. ESLint detected the duplicate `case 3`

File: src/series/graph.serie.line.js [hunk 2]
```diff
@@ -1740,7 +1739,6 @@ class SerieLine extends Serie {
         break;
 
       case false:
-      case 1:
```
Comment (targos) [inline src/series/graph.serie.line.js]: after fixing the first duplicate, this one becomes a duplicate

Comment (NPellet) [inline src/series/graph.serie.line.js]: No, case 1 should come with default

Comment (targos) [inline src/series/graph.serie.line.js]: not sure what you mean

File: src/series/graph.serie.line.js [hunk 3]
```diff
@@ -1703,10 +1702,10 @@ class SerieLine extends Serie {
 
     switch ( this.getStyle( selectionType ).lineStyle ) {
 
-      case 2:
+      case 1:
         return "1, 1";
         break;
-      case 3:
+      case 2:
```
Comment (targos) [inline src/series/graph.serie.line.js]: this case was a duplicate of case 3

File: src/series/graph.serie.line.js (hunk 1 above)
Comment (targos) [inline src/series/graph.serie.line.js]: by fixing case 2, this became a duplicate of case 2

File: src/series/graph.serie.line.js (hunk 2 above)
Comment (targos) [inline src/series/graph.serie.line.js]: by fixing case 1, this became of duplicate of case 1

**Comment by targos** [comment]
ping @NPellet

**Comment by NPellet** [comment]
I rest my case, setLineStyle(1); should be the same as setLineStyle(0) and setLineStyle()

File: src/series/graph.serie.line.js (hunk 1 above)
Comment (targos) [inline src/series/graph.serie.line.js]: > I rest my case, setLineStyle(1); should be the same as setLineStyle(0) and setLineStyle()

Then what should be here ? If it should be 2, what should be on the next case ? If next case should be 3, what should be in the next next case?
I repeat, there is a duplicate case, so the last one (currently the second case 3) is never reached.

**Comment by NPellet** [comment]
Yeah all numbers had to be shifted by one actually. Or move case 3 to an
additional case in the end

2017-02-21 9:04 GMT+01:00 Michaël Zasso <notifications@github.com>:

**Comment by targos** [comment]
Updated. PTAL

**Comment by NPellet** [comment]
Ah no sorry, case 1 goes with case false.
When corrected you can merge

**Pull request merged**